app/assets/javascripts/dialog_field_refresh.js

```diff
@@ -1,16 +1,20 @@
 /* global miqInitSelectPicker miqSelectPickerEvent miqSparkle miqSparkleOn */
 
 var dialogFieldRefresh = {
+  unbindAllPreviousListeners: function() {
+    $(document).off('autoRefresh');
```

Comment:
I like dialog::autoRefresh - or dialogAutoRefresh - just something to make it obvious it's our message.  Thanks a lot for explanation!